graphos/templates/graphos/gchart/base.html

```diff
@@ -2,7 +2,12 @@
 <script type="text/javascript">
   google.setOnLoadCallback(drawChart{{ chart.get_html_id }});
   function drawChart{{ chart.get_html_id }}() {
-    var data = google.visualization.arrayToDataTable({{ chart.get_data_json|safe }});
+    {% if chart.get_chart_type == 'candle_stick' %}
+        var data = google.visualization.arrayToDataTable({{ chart.get_data_json|safe }}, true);
+    {% else %}
+        var data = google.visualization.arrayToDataTable({{ chart.get_data_json|safe }});
+
+    {% endif %}
```

Comment:
@rameshugar I don't think there having an if/else condition in gchart/base.html is a good idea because what if tomorrow we have to add another chart type which expects data in a different format. Then you will have to add another condition like:

    {% elif chart.get_chart_type == 'blah_blah_chart' %}
        var data = google.visualization.arrayToDataTable({{ chart.get_data_json|safe }}, true, something);

So it's better to have a {% block arrayToDataTableIntialiazation %} in base.html and override it in candlestick_chart.html